src/invision/components/movie-popup/movie.popup.js

```diff
@@ -0,0 +1,87 @@
+    this.handleUrlChange = this.handleUrlChange.bind(this);
+    this.handleGenreChange = this.handleGenreChange.bind(this);
+    this.handleOverviewChange = this.handleOverviewChange.bind(this);
+    this.handleRuntimeChange = this.handleRuntimeChange.bind(this);
+  }
+
+  handleTitleChange(event) {
+    this.setState({ title: event.target.value });
+  }
+
+  handleReleaseDateChange(event) {
+    this.setState({ releaseDate: event.target.value });
+  }
+
+  handleUrlChange(event) {
+    this.setState({ url: event.target.value });
+  }
+  handleGenreChange(event) {
+    this.setState({ genre: event.target.value });
+  }
+
+  handleRuntimeChange(event) {
+    this.setState({ runtime: event.target.value });
+  }
+
+  handleOverviewChange(event) {
+    this.setState({ overview: event.target.value });
+  }
+
+  render() {
+    return (
+      <Modal visible={this.props.visible} width="600" effect="fadeInDown" onClickAway={() => this.props.onModalClose(false)}>
+        <div className="add-movie-ctnr">
+          <div className="header-close-btn"><a onClick={() => this.props.onModalClose(false)}>&times;</a></div>
+          <h1>{this.props.title}</h1>
+          <form className="form-ctnr">
+            <div className="form-field">
+              <label htmlFor="title">Title</label><br />
+              <input type="text" placeholder="Title" id="title" value={this.state.title} onChange={this.handleTitleChange} />
+            </div>
+            <div className="form-field">
+              <label htmlFor="title">Release Date</label><br />
+              <input type="date" placeholder="Release Date" id="release-date" value={this.state.releaseDate} onChange={this.handleReleaseDateChange} />
+            </div>
+            <div className="form-field">
+              <label htmlFor="title">Movie URL</label><br />
+              <input type="text" placeholder="Movie URL" id="url" value={this.state.url} onChange={this.handleUrlChange} />
+            </div>
+            <div className="form-field">
+              <label htmlFor="title">GENRE</label><br />
+              <input type="text" placeholder="GENRE" id="genre" value={this.state.genre} onChange={this.handleGenreChange} />
+            </div>
+            <div className="form-field">
+              <label htmlFor="title">OVERVIEW</label><br />
+              <input type="text" placeholder="OVERVIEW" id="overview" value={this.state.overview} onChange={this.handleOverviewChange} />
+            </div>
+            <div className="form-field">
+              <label htmlFor="title">RUNTIME</label><br />
+              <input type="text" placeholder="RUNTIME" id="runtime" value={this.state.runtime} onChange={this.handleRuntimeChange} />
+            </div>
+
+            <div className="form-footer">
+              <button className="primary-btn reset-button" type="reset">Reset</button>
+              <button className="primary-btn" type="submit">Submit</button>
```

Comment:
This button does not iteractive. But it is not so complicated to add or update movie in the array of movies. Please, add some interactivity to your App. I'm talking not about API calls. Update state of the MovieList component, for example.